.buildkite/pipeline.yml

```diff
@@ -66,9 +66,9 @@ steps:
     depends_on: js-bundles
```

Comment:
Good point. Addressed in 83deadc9aed73a6b693f86e471ec66718442af3a

In the future, maybe we could split the build from the upload, to use less macOS agent time. Then again, uploading doesn't take that long and it might be silly to upload to Buildkite, download from Buildkite, and finally upload to S3 🤔

Reply:
> Then again, uploading doesn't take that long and it might be silly to upload to Buildkite, download from Buildkite, and finally upload to S3 🤔

Ahah, @crazytonyli I just read your [comment below](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5769/files/57f9dd1c4233913696578d89194b7527399be189#r1222286309). 😄